lib/builderator/tasks/packer.rb

```diff
@@ -128,6 +128,42 @@ def wait(profile)
         say_status :complete, 'All copied images are available'
       end
 
+      desc 'remote_tag PROFILE', 'Apply existing tags to the AMI in remote AWS accounts'
+      def remote_tag(profile)
+        invoke :configure, [profile], options
+
+        sts_client = Aws::STS::Client.new(region: Config.aws.region)
+        allowed_cred_keys = %w(access_key_id secret_access_key session_token)
```

Comment:
there should only need to be one `sts_client` for each of the images being re-tagged.